lerna.json

```diff
@@ -7,7 +7,9 @@
     "publish": {
       "ignore": [
         "*.test.js",
-        "*.md"
+        "*.vite_test.js",
+        "*.md",
```

Comment:
Not something to hold up this diff; just musing.

Wouldn't it be really cool if when a release is cut, a version of the docs relating to that specific release were put up so that it doesn't change as we merge more things to master?

Reply:
what we really need is versioned docs... I was thinking of submitting a PR to docusaurus to versioning of independent packages within a single documentation project.  I asked them about it and they're interested in it.  I think that's something we should tackle as part of the frontend infra team.

Reply:
We use docusaurus for the KaTeX docs and versioning single packages works well already.